azext_edge/edge/_help.py

```diff
@@ -62,10 +80,30 @@ def load_iotops_help():
         type: command
         short-summary: Evaluate IoT Operations service deployment for health, configuration and usability.
         long-summary: |
-            [Supported service APIs]
-                {COMPAT_MQ_APIS.as_str()}
-                {COMPAT_DATA_PROCESSOR_APIS.as_str()}
-                {COMPAT_LNM_APIS.as_str()}
+            The command by default shows a human friendly _summary_ view of the selected service.
+            More detail can be requested via `--detail-level`.
+
+            {{Supported service APIs}}
+            - {COMPAT_MQ_APIS.as_str()}
+            - {COMPAT_DATA_PROCESSOR_APIS.as_str()}
+            - {COMPAT_LNM_APIS.as_str()}
+
```

Comment:
I realized I forgot to update the help file for OPCUA check if this release includes it, I can create a pr to address